helix-view/src/view.rs

```diff
@@ -66,24 +66,32 @@ pub struct View {
     pub jumps: JumpList,
     /// the last accessed file before the current one
     pub last_accessed_doc: Option<DocumentId>,
+    pub config: crate::editor::Config,
 }
 
 impl View {
-    pub fn new(doc: DocumentId) -> Self {
+    pub fn new(doc: DocumentId, config: crate::editor::Config) -> Self {
```

Comment:
I think the config should be shared, best without using `RefCell<Mutex<Config>>` for https://github.com/helix-editor/helix/pull/798